sdk/core/Azure.Core/src/Azure.Core.props

```diff
@@ -9,4 +9,8 @@
   <ItemGroup Condition="'$(UseProjectReferenceToAzureCore)'!='true'">
     <PackageReference Include="Azure.Core" Version="1.0.0-preview.5" />
   </ItemGroup>
+
+  <ItemGroup>
+      <Compile Include="$(MSBuildThisFileDirectory)/Shared/*.cs" />
```

Comment:
We should probably make projects opt-in to individual source files instead of always including all of them. We don't want to ship a bunch of shared copies of code that isn't even used in a library.